packages/typespec-azure-resource-manager/lib/operations.tsp

```diff
@@ -707,14 +707,15 @@ op checkLocalNameAvailability<
 @action
 @armResourceCollectionAction
 op ArmProviderActionSync<
-  Request extends {} | void = void,
+  Request extends {} = {},
```

Comment:
I am thinking we may need to have something like `ArmProviderActionNoRequestBody` kind of operations. Since it closely mirror Actions (combination of Get | Post,  No Req body | No Res body), we may need a more flexible story here. Let's discuss when you are back.